pkg/webhook/appFilterCheck.go

```diff
@@ -27,6 +27,47 @@ import (
 	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
 )
 
+func (wh *webhook) ValidateChaosExperimentsConfigMaps(chaosEngine *v1alpha1.ChaosEngine) error {
+	configMaps, err := wh.kubeClient.CoreV1().ConfigMaps(chaosEngine.Spec.Appinfo.Appns).List(metav1.ListOptions{
```

Comment:
We could simply use a `Get` call rather than `List` here.
So, what we can do here is, directly loop over the `ChaosExperiment` listed in `ChaosEngine`, and check the if the we are able to get the configmap listed in application namespace.
WDYT?

Reply:
You are right. Implementation is much simpler now.

Reply:
Looks great, I think we can get this PR out of Draft now, and ask review form others too.

Reply:
Great. I will have a look into #14 in the meantime, as the implementation should be similar.